src/table/src/metadata.rs

```diff
@@ -269,7 +269,11 @@ impl TableMeta {
                     self.change_column_skipping_index_options(table_name, column_name, None)
                 }
             },
-            AlterKind::DropDefaults { names } => self.drop_defaults(table_name, names),
+            AlterKind::UnsetDefault(name) => self.unset_default(table_name, name.to_owned()),
+            AlterKind::SetDefault {
+                column_name,
+                default_constraint,
+            } => self.set_default(table_name, column_name.clone(), default_constraint.clone()),
```

Comment:
Modify `set_default` and `unset_default` to accept references to the column name and constraint. Prefer passing references to functions and clone only when necessary.